catkin_virtualenv/cmake/catkin_generate_virtualenv.cmake

```diff
@@ -58,8 +58,7 @@ function(catkin_generate_virtualenv)
     OUTPUT_STRIP_TRAILING_WHITESPACE
   )
 
-  set(PYTHON_VERSION_MAJOR ${ARG_PYTHON_VERSION_MAJOR})
```

Comment:
not sure what the goal of this change is, but it breaks the nosetest logic further down.

Reply:
does it cause test failures on travis?
it seems `PYTHON_VERSION_MAJOR` is no need to set.

Reply:
kindly ping @paulbovbel